contents/ARCME.oscmeta

```diff
@@ -0,0 +1,15 @@
+{
+    "information": {
+        "name": "Any Region Changer: ModMii Edition",
+        "author": "Various, bushing, tona, scooby74029, TheShadowEevee, XFlak",
```

Comment:
Is the author field in the meta.xml this long? I think it could be trimmed down to just "Various".